config.md

```diff
@@ -8,14 +8,17 @@ This includes the process to run, environment variables to inject, sandboxing fe
 
 Below is a detailed description of each field defined in the configuration format.
 
-## Manifest version
+## Specification version
 
-* **`version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OCF specification with which the container bundle complies. The Open Container spec follows semantic versioning and retains forward and backward compatibility within major versions. For example, if an implementation is compliant with version 1.0.1 of the spec, it is compatible with the complete 1.x series. NOTE that there is no guarantee for forward or backward compatibility for version 0.x.
+* **`spec_version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OpenContainer specification with which the bundle complies.
```

Comment:
+1 to more specific name.

On Fri, Jan 15, 2016 at 12:25 AM, W. Trevor King <notifications@github.com>
wrote:

> In config.md
> <https://github.com/opencontainers/specs/pull/309#discussion_r49801534>:
>
> >
> > -* **`version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OCF specification with which the container bundle complies. The Open Container spec follows semantic versioning and retains forward and backward compatibility within major versions. For example, if an implementation is compliant with version 1.0.1 of the spec, it is compatible with the complete 1.x series. NOTE that there is no guarantee for forward or backward compatibility for version 0.x.
> > +* **`spec_version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OpenContainer specification with which the bundle complies.
>
> On Thu, Jan 14, 2016 at 02:37:10PM -0800, Vincent Batts wrote: > +*
> **`spec_version`** (string, required) must be in [SemVer v2.0.0](
> http://semver.org/spec/v2.0.0.html) format and specifies the version of
> the OpenContainer specification with which the bundle complies. i already
> pushed a change for this. PTAL
> 4df6c3b
> <https://github.com/opencontainers/specs/commit/4df6c3bb5622113268878775883031bbd4ee1e16>
> (the current tip) has specVersion in config.go but spec_version in
> config.md (twice). Are we sure we don't want a more specific name [1,2]?
> [1]: #309 (comment)
> <https://github.com/opencontainers/specs/pull/309#discussion_r49667293>
> [2]: #110 (comment)
> <https://github.com/opencontainers/specs/issues/110#issuecomment-171470005>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/309/files#r49801534>.
>